taxcalc/calculate.py

```diff
@@ -160,6 +160,12 @@ def increment_year(self):
         self.policy.set_year(self.policy.current_year + 1)
         self.behavior.set_year(self.policy.current_year)
 
+    def year_increment(self, new_current_year):
+        iteration = new_current_year - self.policy.current_year
```

Comment:
I think this function needs:
- a doc string on what it does
- a check that `iteration` is non-zero, with an exception if `new_current_year` is less than `self.policy.current_year`

The PR also needs:

 - a test on expected behavior in the regular case (properly advancing the year)
 - a test to make sure the function raises an exception when the year is less than the current year

I would also suggest a better name, such as `advance_to_year`, which indicates that the argument is a year.